packages/cli/scripts/preinstall.js

```diff
@@ -1,21 +1,33 @@
+const process = require('node:process')
 // @ts-check
 const { drawBox } = require('@prisma/internals/dist/utils/drawBox')
 
 export function main() {
-  // process.version (e.g. `v16.0.0`)
-  printMessageAndExitIfUnsupportedNodeVersion(process.version)
+  // Node.js version, without the `v` prefix (e.g. `16.13.0`)
+  const nodeVersion = process.versions.node
```

Comment:
We can keep using `process.version`, it's going to be the safest one to use and it doesn't bring value of changing this.
```suggestion
  const nodeVersion = process.versions
```

Reply:
`process.versions` exists since Node.js 9.0.0, and arguably `process.versions.node` is more ergonomic and explicit than `process.version.slice(1)`, but discussing about this nit is of little value imho, so I don't have a strong opinion.

Reply:
I've removed `process.versions.node` in favor of `process.version.slice(1)`.